.github/CODEOWNERS

```diff
@@ -232,7 +232,7 @@
 # ServiceOwners:                                                   @conhua @mengaims @juaduan @moreOver0
 
 # PRLabel: %Cognitive - Health Insights
-/sdk/healthinsights/                                               @asaflevi-ms
+/sdk/healthinsights/                                               @asaflevi-ms @tomsft @koen-mertens
```

Comment:
I think you can add your names under radiology insights and not entire health insights SDKs.